neqo-transport/src/connection/mod.rs

```diff
@@ -820,6 +836,11 @@ impl Connection {
             }
         }
 
+        if let Some(t) = self.release_resumption_token_timer {
+            qtrace!([self], "Resumption token event timer {:?}", t);
+            delays.push(t);
+        }
```

Comment:
So I think I might not have been clear.  I agree with you about not adding this to the set of delays here.  But, I think that it is worth adding a comment here explaining that you aren't considering that timer as it is not important enough to force the application to set a timer for (and you expect other activity to drive it).

It's not that this needs to be considered, it's that we need to establish a pattern whereby you update both process_timeout and next_delay together when adding new timers.